apps/projects/app/components/Card/Issue.js

```diff
@@ -9,6 +9,7 @@ import {
   Checkbox,
   ContextMenu,
   ContextMenuItem,
+  Viewport,
```

Comment:
`Viewport` AND `ContextMenuItem` are not being used, can be removed

Reply:
sure.